### PR TITLE
InstructorHomePageUiTest failing frequently #4301

### LINKED ISSUE #4301: InstructorHomePageUiTest failing frequently
Due to element not found, element no longer attached to DOM

### PR DESCRIPTION
**Pull request by LowWeiLin**
Fixes #4301

### CODE REVIEW

File: src/test/java/teammates/test/pageobjects/InstructorHomePage.java [hunk 1]
```diff
@@ -280,10 +280,16 @@ public Url getArchiveCourseLink(String courseId){
     }
     
     private WebElement getSessionLinkInRow(String elementClassNamePrefix, int rowId){
+        waitForElementPresence(By.id("session" + rowId));
+        By.id("session" + rowId);
```
Comment (junhaoyap) [inline src/test/java/teammates/test/pageobjects/InstructorHomePage.java]: Seems like an extraneous line to me?

File: src/test/java/teammates/test/pageobjects/InstructorHomePage.java [hunk 2]
```diff
@@ -280,10 +280,16 @@ public Url getArchiveCourseLink(String courseId){
...
+        waitForElementPresence(By.className(elementClassNamePrefix));
         return browser.driver.findElement(By.id("session" + rowId)).findElement(By.className(elementClassNamePrefix));
     }
     
     private WebElement getCourseLinkInRow(String elementClassNamePrefix, int rowId){
+        waitForElementPresence(By.id("course-" + rowId));
+        By.id("course-" + rowId);
```
Comment (junhaoyap) [inline src/test/java/teammates/test/pageobjects/InstructorHomePage.java]: ditto

File: src/test/java/teammates/test/pageobjects/InstructorHomePage.java (hunk 1 above)
Comment (LowWeiLin) [inline src/test/java/teammates/test/pageobjects/InstructorHomePage.java]: Right. Not thinking properly.

Comment (junhaoyap) [inline src/test/java/teammates/test/pageobjects/InstructorHomePage.java]: You worked on it so long, I'm sure you were too tired, take a break!

**Pull request merged**